[2023-06-09T14:54:10.932+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2023-06-07T00:00:00+00:00 [queued]>
[2023-06-09T14:54:10.942+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2023-06-07T00:00:00+00:00 [queued]>
[2023-06-09T14:54:10.947+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 7
[2023-06-09T14:54:10.996+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-06-07 00:00:00+00:00
[2023-06-09T14:54:11.006+0000] {standard_task_runner.py:57} INFO - Started process 1937 to run task
[2023-06-09T14:54:11.014+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2023-06-07T00:00:00+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpxl1g9g4k']
[2023-06-09T14:54:11.027+0000] {standard_task_runner.py:85} INFO - Job 61: Subtask insert_into_table
[2023-06-09T14:54:11.157+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2023-06-07T00:00:00+00:00 [running]> on host 94560a00d375
[2023-06-09T14:54:11.233+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='znawawi' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-06-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-07T00:00:00+00:00'
[2023-06-09T14:54:11.235+0000] {sql.py:262} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-06-07', 'dag_with_postgres_operator_v04')
        
[2023-06-09T14:54:11.244+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-09T14:54:11.293+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-09T14:54:11.305+0000] {sql.py:375} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-06-07', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-06-09T14:54:11.308+0000] {sql.py:384} INFO - Rows affected: 1
[2023-06-09T14:54:11.321+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20230607T000000, start_date=20230609T145410, end_date=20230609T145411
[2023-06-09T14:54:11.361+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-09T14:54:11.376+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
